apps/condo/domains/user/schema/SignInUserService.js

```diff
@@ -0,0 +1,233 @@
+/**
+ * Generated by `createservice user.SignInUserService --type mutations`
+ */
+
+const get = require('lodash/get')
+
+const conf = require('@open-condo/config')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')
+const { GQLCustomSchema, find, getSchemaCtx, getById } = require('@open-condo/keystone/schema')
+const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
+const { i18n } = require('@open-condo/locales/loader')
+
+const { NOT_FOUND } = require('@condo/domains/common/constants/errors')
+const { DV_VERSION_MISMATCH, WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { normalizeEmail } = require('@condo/domains/common/utils/mail')
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/user/access/SignInUserService')
+const { SERVICE, STAFF, RESIDENT } = require('@condo/domains/user/constants/common')
+const { ConfirmPhoneAction, User } = require('@condo/domains/user/utils/serverSchema')
+
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    REQUIRED_USER_DATA_IS_MISSING: (missingFields = [], missingFieldsForUser = []) => ({
```

Comment:
Usually, I have 2 ways to tell user, what's wrong:

1. Show notification with `messageForUser`
2. Display `messageForUser` under some field of submitted form (error.type to form field mapping)

Which of them should I use on this scenario?

Reply:
Updated this error. 
Now one specific field is specified - we can use it for mapping in form